spec/features/admin_spec.rb

```diff
@@ -0,0 +1,32 @@
+# spec/features/admin_spec.rb
+require 'spec_helper'
+
+describe "Admin Panel" do
```

Comment:
Since you are testing that the controller handles the admin stuff, I don't think there is any need for this test anymore.